docs/index.md

```diff
@@ -116,6 +117,30 @@ Passing a token to the verification endpoint will return a 200 response and the
 $ curl -X POST -H "Content-Type: application/json" -d '{"token":"<EXISTING_TOKEN>"}' http://localhost:8000/api-token-verify/
 ```
 
+## Blacklist Token
+If `JWT_ENABLE_BLACKLIST` is True, and `rest_framework_jwt.blacklist` has been added to `INSTALLED_APPS`, tokens can be made invalid (prior to expiration) by blacklisting them. More information on the concept of JTI (JWT ID) and blacklisting tokens can be read [here](http://self-issued.info/docs/draft-ietf-oauth-json-web-token.html#jtiDef) and [here](https://auth0.com/blog/2015/03/10/blacklist-json-web-token-api-keys/).
+
+This package comes with a default implementation that stores the blacklisted tokens in the configured django database and includes an admin integration.
+
+To use this feature, add a URL like so:
+
+```python
+    url(r'^api-token-blacklist/', 'rest_framework_jwt.views.blacklist_jwt_token'),
```

Comment:
yes `BlacklistJSONWebToken` is under the blacklist app but the actual checks for blacklisted tokens still occur inside the authenticate method in authentication.py and inside _check_payload in serializers.VerificationSerializer.

I agree we can drop that setting but I ran into an issue where the unit tests break. I changed the check within above files to be:
```
        if 'rest_framework_jwt.blacklist' in settings.INSTALLED_APPS:
```
But in the unit tests previously I was settings `JWT_ENABLE_BLACKLIST = True` for the blacklist tests. I removed that line and get an error now saying the blacklist app is not installed (error message from check in blacklist.serializers.BlacklistJSONWebTokenSerializer)

The blacklist app is installed in tests/conftest.py, how can I get it to work with the unit test?

Previously this is what a test looked like but now removed line for `JWT_ENABLE_BLACKLIST`:
```
class BlacklistJSONWebTokenTests(TokenTestCase):

    def test_blacklist_jwt_successful_blacklist_enabled(self):
        api_settings.JWT_ENABLE_BLACKLIST = True

        client = APIClient(enforce_csrf_checks=True)

        user = User.objects.create_user(
            email='jsmith@example.com', username='jsmith', password='password')

        token = self.create_token(user)

        # Handle blacklisting the token.
        response = client.post('/auth-token-blacklist/', {'token': token},
                               format='json')

        msg = 'Token successfully blacklisted.'

        self.assertEqual(response.data['message'], msg)
```